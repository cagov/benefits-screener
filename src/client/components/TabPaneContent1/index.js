import { Trans, useTranslation } from "react-i18next";
import React from "react";

function TabPaneContent1() {
  const { t } = useTranslation();

  return (
    <div>
      <h3>
        <Trans t={t} i18nKey="tab1.header10" />
      </h3>

      <p>
        <Trans t={t} i18nKey="tab1.p10" />
      </p>

      <ul>
        <li>
          <Trans t={t} i18nKey="tab1.list10-item10" />
        </li>
        <li>
          <Trans t={t} i18nKey="tab1.list10-item20" />
        </li>
        <li>
          <Trans t={t} i18nKey="tab1.list10-item30" />
        </li>
      </ul>

      <p>
        <Trans t={t} i18nKey="tab1.p20" />{" "}
      </p>

      <h3>
        <Trans t={t} i18nKey="tab1.header20" />
      </h3>

      <p>
        <Trans t={t} i18nKey="tab1.p30" />
      </p>

      <ul>
        <li>
          <Trans t={t} i18nKey="tab1.list20-item10" />
        </li>
        <li>
          <Trans t={t} i18nKey="tab1.list20-item20" />
        </li>
      </ul>
      <h3>
        <Trans t={t} i18nKey="tab1.header30" />
      </h3>
      <ul>
        <li>
          <Trans t={t} i18nKey="tab1.list30-item10" />
        </li>
        <li>
          <Trans t={t} i18nKey="tab1.list30-item20" />
        </li>
      </ul>
    </div>
  );
}

export default TabPaneContent1;
