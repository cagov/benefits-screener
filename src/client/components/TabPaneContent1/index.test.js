import React from "react";
import TabPaneContent1 from "./index";
import { shallow } from "enzyme";

describe("<TabPaneContent1 />", () => {
  it("renders content pane", async () => {
    const wrapper = shallow(<TabPaneContent1 />);

    expect(wrapper).toMatchSnapshot();
  });
});
