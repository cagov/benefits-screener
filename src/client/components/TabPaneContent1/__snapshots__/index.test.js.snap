// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabPaneContent1 /> renders content pane 1`] = `
<div>
  <h3>
    Proof of citizenship or authorization to work in the United States
  </h3>
  <p>
    You will need 
    <strong>
      all
    </strong>
     of following:
  </p>
  <ul>
    <li>
      Social Security number
    </li>
    <li>
      California driver license or ID card
    </li>
    <li>
      Proof of US citizenship, or a green card, or a visa that allows you to work in the US, or an Alien Registration Number
    </li>
  </ul>
  <p>
    You must be authorized to work in the US to receive unemployment benefits. If you are not a US citizen, have information from your employment authorization document ready.
  </p>
  <h3>
    Information about your work history
  </h3>
  <p>
    Your work history affects the amount of benefits you receive each week. Submitting accurate information helps us process your claim faster. You will need:
  </p>
  <ul>
    <li>
      <strong>
        Work history from the last 18 months.
      </strong>
       This includes the name of any companies as they appear on your paycheck, the dates of employment, hours worked per week, the total wages you earned before taxes were taken out (gross wages), hourly rate of pay, and the reason you are no longer working.
    </li>
    <li>
      <strong>
        Specific information from your last employer or company.
      </strong>
       
      This includes your employer’s name, mailing address, phone number, supervisor’s name, total wages you earned before taxes were taken out (gross wages) for the last week you worked, and the reason for your unemployment.
    </li>
  </ul>
  <h3>
    Federal and military documents
  </h3>
  <ul>
    <li>
      <strong>
        If you are a former federal employee,
      </strong>
       have your
       
      <i>
        Notice to Federal Employees About Unemployment Insurance
      </i>
       
      (Standard Form 8) ready.
    </li>
    <li>
      <strong>
        If you served in the military in the last 18 months,
      </strong>
       
      have your 
      <i>
        Certificate of Release or Discharge from Active Duty
      </i>
       
      (DD 214) ready.
    </li>
  </ul>
</div>
`;
